app.R

```diff
@@ -947,16 +967,40 @@ server <- function(input, output, session) {
   
   # display stats for least concern species
   output$stats <- DT::renderDataTable({
-    if (! is.null(values$statistics)) {
-      filter(values$statistics,
-             EOO >= eooValue(),
-             AOO >= aooValue(),
-             RecordCount >= recordsValue(),
-             TDWGCount >= tdwgValue())
-      }
       
-    }, 
-    options = list(pageLength = 5))
+      req(values$statistics)
+      
+      values$statistics <- mutate(values$statistics,
+                                  leastConcern=EOO >= eooValue() & 
+                                    AOO >= aooValue() & 
+                                    RecordCount >= recordsValue() & 
+                                    TDWGCount >= tdwgValue())
+    
+      datatable(arrange(values$statistics, desc(leastConcern)),
+                options=list(pageLength=5)) %>%
+        formatStyle("leastConcern",
+                    target="row",
+                    color=styleEqual(c(1, NA), c("green", "red")))
+    })
+  
+  output$stats_warnings <- renderPrint({
+    req(values$statistics)
+    
+    if ("leastConcern" %in% colnames(values$statistics)) {
+      n_least_concern <- sum(values$statistics$leastConcern, na.rm=TRUE)
+      n_warnings <- sum(is.na(values$statistics$leastConcern))
+      n_total <- nrow(values$statistics)
+      
+      cat(
+        sprintf("%d species considered", n_total),
+        sprintf("%d with warnings", n_warnings),
+        sprintf("%d identified as least concern", n_least_concern),
```

Comment:
Could maybe add something like 'identified as Least Concern based on user-defined thresholds' Or something else to highlight that 'we' are not saying LC, the user is deciding that based on the slider thresholds.